parser.go

```diff
@@ -96,7 +97,7 @@ func Parse(arquivos []string, chave_coleta string) (*coleta.FolhaDePagamento, er
 		folha = append(folha, contra_cheque...)
 	}
 	if parseErr {
-		return &coleta.FolhaDePagamento{ContraCheque: folha}, fmt.Errorf("parse error")
+		return &coleta.FolhaDePagamento{ContraCheque: folha}, status.NewError(status.Unknown, fmt.Errorf("parse error"))
```

Comment:
Dar mais informação aqui
```suggestion
		return &coleta.FolhaDePagamento{ContraCheque: folha}, status.NewError(status.Unknown, fmt.Errorf("parsing error: %w", parseErr))
```